pandas/core/generic.py

```diff
@@ -2357,8 +2357,8 @@ def to_hdf(
         nan_rep=None,
         dropna: Optional[bool_t] = None,
         data_columns: Optional[List[str]] = None,
-        errors: str = "strict",
         encoding: str = "UTF-8",
+        errors: str = "strict",
```

Comment:
is there a reason you changed this?

Reply:
This order is in line with the order of arguments in `open()` of the Python standard library. I thought it would be nice to have a consistent order of arguments.

Reply:
Let’s revert, to not break users passing them by position.

Reply:
You are kidding right, this is the 13'th argument of this method. Nobody in their right minds would *ever* use this one as a positional argument. (masochists excluded)

Reply:
No, I'm not kidding.

Reply:
Whatever, I'll revert